tools/api-simulations/lib/src/Network/Wire/Simulations.hs

```diff
@@ -65,11 +65,11 @@ prepareConv g
             "Missing 1-1 conversation between: " <>
                 Text.concat (Text.pack . show . botId <$> [a, b])
     | otherwise = do
-        let (a : b : c : cs) = g
+        let (a : bs) = g
         connectIfNeeded g
-        let cIds = botId <$> list1 c cs
```

Comment:
Rewriting this piece of code a bit because the interface of `createConv` has been changed